corrector.rb

```diff
@@ -0,0 +1,15 @@
+class Corrector
+  def initialize() end
```

Comment:
- [ ] **[OPTIONAL]** In keeping with the YAGNI (You Ain't Gonna Need It) rule, I would recommend you remove this **initialize** method since it isn't used. In Ruby class definitions you must not define the **initialize** method if you won't use it.